src/meta/buildTags.tsx

```diff
@@ -36,6 +36,28 @@ const buildTags = (config: BuildTagsParams) => {
     defaults.nofollow ||
     config.dangerouslySetAllPagesToNoFollow;
 
+  let robotsParams = '';
+  if (config.additionalRobotsProps) {
+    const nosnippet = config.additionalRobotsProps.nosnippet;
+    const maxSnippet = config.additionalRobotsProps.maxSnippet;
+    const maxImagePreview = config.additionalRobotsProps.maxImagePreview;
+    const maxVideoPreview = config.additionalRobotsProps.maxVideoPreview;
+    const noarchive = config.additionalRobotsProps.noarchive;
+    const unavailableAfter = config.additionalRobotsProps.unavailableAfter;
+    const noimageindex = config.additionalRobotsProps.noimageindex;
+    const notranslate = config.additionalRobotsProps.notranslate;
```

Comment:
We could change this to use destructing here instead? 

```
const { nosnippet, maxSnippet, etc } = config.additionalRobotsProps;
```

Reply:
Thanks for the feedback, for sure I'll be on that this weekend